R/theme-defaults.r

```diff
@@ -134,7 +134,7 @@ theme_grey <- function(base_size = 11, base_family = "") {
     panel.background =   element_rect(fill = "grey92", colour = NA),
     panel.border =       element_blank(),
     panel.grid.major =   element_line(colour = "white"),
-    panel.grid.minor =   element_line(colour = "white", size = 0.25),
+    panel.grid.minor =   element_line(colour = "white", size = rel(0.25)),
```

Comment:
I noticed that these actually change the values of the width as the `base_size` (as far as I can tell) is set to 0.5 by `line`.  I think that means all the values inside `rel()` should be doubled.

It also suggests that before merging this PR we should make some basic visual tests for the themes.